package.json

```diff
@@ -33,7 +33,7 @@
     "commander":                        "^2.4.0",
     "cliff":                            "^0.1.10",
     "browserify":                       "^9.0.3",
-    "mocha-phantomjs":                  "^3.5.3",
+    "mocha-phantomjs":                  "3.5.3",
```

Comment:
We can't have comments in json files... I'm not too worried.. hopefully mocha-phantomjs will be fixed eventually... Or we should move to karma or something more stable...